src/styles/index.scss

```diff
@@ -1,3 +1,216 @@
+  font-weight: 500;
+  text-align: left;
+  text-transform: uppercase;
+  color: black;
+  text-decoration: none;
+}
+
+.link:hover {
+  color: var(--color-blue);
+}
+
+.nav__list li:first-child .link {
+  margin-left: 0;
+}
+
+.link--is_active {
+  color: var(--color-blue);
+}
+
+.link.is-active::after {
+  content: '';
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background: var(--color-blue);
+}
+
+$color-main: #060b35;
+$color-secondary: #616070;
+$button-main: #008bb0;
+$button-hover: white;
+$font-family: Roboto, sans-serif;
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+
+  &__image {
+    padding: 32px 19px 40px;
+  }
+
+  &__name {
+    font-family: $font-family;
+    font-size: 12px;
+    font-weight: 500;
+    color: $color-main;
+    padding-left: 16px;
+    padding-right: 16px;
+    padding-bottom: 4px;
+  }
+
+  &__code {
+    font-family: $font-family;
+    font-size: 10px;
+    width: 98px;
+    height: 14px;
+    color: $color-secondary;
+    padding-left: 16px;
+    padding-bottom: 16px;
+  }
+}
+
+.card__stars {
+  display: flex;
+  position: relative;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  height: 16px;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-self: center;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+  }
+
+  &--1 .stars__star:nth-child(1),
+  &--2 .stars__star:nth-child(-n + 2),
+  &--3 .stars__star:nth-child(-n + 3),
+  &--4 .stars__star:nth-child(-n + 4),
+  &--5 .stars__star:nth-child(-n + 5) {
+    background-image: url(../images/star-active.svg);
+  }
+}
+
+.card__reviews {
+  font-family: $font-family;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: $color-main;
+}
+
+.card__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-bottom: 16px;
+
+  &-label {
+    font-family: $font-family;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+    color: $color-secondary;
+  }
+
+  &-amount {
+    font-family: $font-family;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+    color: $color-main;
+  }
+}
+
+.card__button {
+  margin-left: 16px;
+  margin-bottom: 16px;
+  margin-right: 16px;
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  background: $button-main;
+
+  &-link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-family: Roboto, sans-serif;
```

Comment:
The font-family is hardcoded as 'Roboto, sans-serif' here. Consider using the $font-family variable for consistency and easier maintenance.